README.md

```diff
@@ -112,12 +112,26 @@ systemctl restart openvswitch
 systemctl enable --now NetworkManager openvswitch
 ```
 
-### Install nmstate from PyPi
+### Install nmstate from PyPi (Python 2)
 ```shell
 yum -y install python2-pip
 pip uninstall -y nmstate; pip install nmstate
 ```
 
+### Install nmstate from PyPi on RHEL 8
+
+Minimal Nmstate installation:
+``` shell
+# install binary dependencies; The development packages are needed to build
+# python-dbus which is improperly packaged on RHEL 8
```

Comment:
Please refer to https://bugzilla.redhat.com/show_bug.cgi?id=1717400 for RHEL 8 dbus-python issue.

Reply:
Added a link to the Bug.